x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -72,9 +73,8 @@ function getSpanTypes(span: Span) {
   };
 }
 
-const TagName = styled.div`
-  font-weight: bold;
-`;
+const getSpanHttpStatusCode = (span: Span): number | undefined =>
+  get(span, 'span.http.response.status_code');
```

Comment:
I think it might be better to use `idx` than `get` because it's aware of the type and you won't have to explicitly define the function return type.